### PR TITLE
buildjre8: improvements

### PR DESCRIPTION
**Pull request by fukiame**
- fixes to some small build issues
- update github actions runner and the actions used

### CODE REVIEW

**Comment by fukiame** [comment]
Screenshot_20240115-164325_PojavLauncher (Minecraft_ Java Edition for Android)
the actions at the forked repo [ran fine](https://github.com/TelegramAt25/android-openjdk-build-multiarch/actions/runs/7526720011), and the build artifacts ran without issues
this pr should now be ready for review

**Comment by artdeell** [comment]
> Screenshot_20240115-164325_PojavLauncher (Minecraft_ Java Edition for Android) the actions at the forked repo [ran fine](https://github.com/TelegramAt25/android-openjdk-build-multiarch/actions/runs/7526720011), and the build artifacts ran without issues this pr should now be ready for review

LGTM, but have you tested the jre8-pojav automatic installation package? (needs to be put into pojav during build)

**Comment by fukiame** [comment]
> > Screenshot_20240115-164325_PojavLauncher (Minecraft_ Java Edition for Android) the actions at the forked repo [ran fine](https://github.com/TelegramAt25/android-openjdk-build-multiarch/actions/runs/7526720011), and the build artifacts ran without issues this pr should now be ready for review
> 
> LGTM, but have you tested the jre8-pojav automatic installation package? (needs to be put into pojav during build)

will do that now, thanks for the comment

**Comment by fukiame** [comment]
Screenshot_20240115-171056_PojavLauncher (Minecraft_ Java Edition for Android)
[test commit used to download the forked artifact](https://github.com/TelegramAt25/PojavLauncher/commit/1586fd0ff0a468a4458f1d4cd3b69527e0ff20f9)
[github actions link of the commit above](https://github.com/TelegramAt25/PojavLauncher/actions/runs/7527216552)

the resulting app-debug apk works fine

> > > Screenshot_20240115-164325_PojavLauncher (Minecraft_ Java Edition for Android) the actions at the forked repo [ran fine](https://github.com/TelegramAt25/android-openjdk-build-multiarch/actions/runs/7526720011), and the build artifacts ran without issues this pr should now be ready for review
> > 
> > 
> > LGTM, but have you tested the jre8-pojav automatic installation package? (needs to be put into pojav during build)
> 
> will do that now, thanks for the comment

**Comment by fukiame** [comment]
> > Screenshot_20240115-164325_PojavLauncher (Minecraft_ Java Edition for Android) the actions at the forked repo [ran fine](https://github.com/TelegramAt25/android-openjdk-build-multiarch/actions/runs/7526720011), and the build artifacts ran without issues this pr should now be ready for review
> 
> LGTM, but have you tested the jre8-pojav automatic installation package? (needs to be put into pojav during build)

is there any other tests that should be done?

**Comment by artdeell** [comment]
No, no need. Running on our actions and merging.

**Comment by artdeell** [comment]
Will merge after i get a second opinion from @Mathias-Boulay

File: tarjdk.sh
Comment (Mathias-Boulay) [inline tarjdk.sh:7]: Why not use the 2.2.**1** release ?

**Review by Mathias-Boulay — CHANGES_REQUESTED** [review]
At least one edit has to be made. Overall a nice PR to keep things up to date.
It guess only the NDK is outdated now

File: tarjdk.sh
Comment (fukiame) [reply tarjdk.sh:7]: > Why not use the 2.2.**1** release ?

that release introduced [a commit](https://github.com/termux/termux-elf-cleaner/commit/9578f2c4bcb3034acecab452f7180e63b51491da) i tried to avoid, as per [the commit message here](https://github.com/PojavLauncherTeam/android-openjdk-build-multiarch/pull/19/commits/ebfd5b7a6a5483070e7a432ac5caf9c1eb4ba61a)

going further causes [this](https://github.com/fukiame/android-openjdk-build-aarch64/actions/runs/7525768725/job/20482664536#step:4:4356) and the build fails

Comment (fukiame) [reply tarjdk.sh:7]: [termux-elf-cleaner 2.2.1 release notes](https://github.com/termux/termux-elf-cleaner/releases/tag/v2.2.1)

Comment (Mathias-Boulay) [reply tarjdk.sh:7]: Fair enough, I'll let this slide for now, given the other priorities

**Review by Mathias-Boulay — APPROVED** [review]
Approved for **squash** merge

**Comment by Mathias-Boulay** [comment]
@fukiame Feel free to perform a similar update on the buildjre17 if you have some time :smile: 
If you have even more time, we are looking to upgrade the NDK version of both buildjre17 and buildjre8 branch.

**Comment by fukiame** [comment]
> @fukiame Feel free to perform a similar update on the buildjre17 if you have some time 😄 If you have even more time, we are looking to upgrade the NDK version of both buildjre17 and buildjre8 branch.

will do the update on buildjre17, and ig buildjre21 too
regarding the NDK... dont think i can do it, skill issue, will try to if i have the time though

**Pull request merged**
